xdem/coreg.py

```diff
@@ -756,43 +756,43 @@ def _apply_pts_func(self, coords: np.ndarray) -> np.ndarray:
         raise NotImplementedError("This should have been implemented by subclassing")
 
 
-class BiasCorr(Coreg):
+class VerticalShift(Coreg):
```

Comment:
Should this not be moved to `biascorr.py`?

Reply:
On a logical sense, it should yes. On the other hand, I wonder if people won't get lost in having two different modules that deal with relatively similar issues. The along/across-track corrections have a very different use case, but a vertical shift or deramping is most often used during coregistration...
I don't have a strict opinion on this, just bringing up the issue!

Reply:
A `VerticalShift` is an affine transformation, so we should keep it in `coreg.py`... no?